README.md

```diff
@@ -118,14 +118,34 @@ will allow you to use `<big-red-text content="Hello!"></big-red-text>` in your `
 
 It is highly suggested that you write Yolk with JSX. This is achieved using the [Babel transpiler](http://babeljs.io/). You should configure the `jsxPragma` option for Babel either in `.babelrc` or in `package.json`:
 
-`.babelrc`:
+__Babel 6+__
+
+The release of Babel 6.0 requires a seperate plugin to be installed called ```babel-plugin-transform-react-jsx``` for transforming. Use the below dependenies and plugin configuration:
+
+`package.json`:
 
 ```json
-{
-  "jsxPragma": "Yolk.createElement"
+{  
+"devDependencies": {
+    "babel-core": "^6.0.14",
+    "babel-plugin-transform-react-jsx": "^6.0.14",
+    "babel-preset-es2015": "^6.0.14"
+  },
+  "babel": {
+    "plugins": [
```

Comment:
It could be, but I think it generally starts bloating the README to list both babelrc and package.json methods, when they are identical. I choose to list just the former as it seems to be the more used method in the community afaik.